index.json

```diff
@@ -9,5 +9,7 @@
     "trailingComma": "none",
     "bracketSpacing": true,
     "arrowParens": "avoid",
-    "endOfLine": "lf"
-}
\ No newline at end of file
+    "endOfLine": "lf",
+    "svelteSortOrder": "options-scripts-styles-markup",
```

Comment:
out of curiosity, why do we need this? I mean, why not use the default svelte option (`options-scripts-markup-styles`)? It's [here](https://github.com/sveltejs/prettier-plugin-svelte?tab=readme-ov-file#options)

Reply:
Good question!
This is the convention we used since the beginning.
And now, from a recent update the IDE starts to reorder the blocks in each touched file, which is very annoying.

See: https://github.com/oat-sa/tao-deliver-fe/blob/develop/prettier.config.js#L23

Reply:
Well, as a company we have 4 years and a few hundred components behind us where the `scripts-styles-markup` order (the original default in Svelte 3) has been used, and I guess (?) we like it. But I'm not aware if we have started doing it the new way in some projects.

Reply:
Thanks for sharing the context, sounds good! I was not aware of this. We might need to reformat projects like control center, but that should be an easy commit.

About the `svelteBracketNewLine` option, [it was deprecated in Prettier 2.5.0](https://github.com/sveltejs/prettier-plugin-svelte?tab=readme-ov-file#svelte-bracket-new-line). They [suggest to use the new Prettier option](https://prettier.io/docs/en/options.html#bracket-line) (not Svelte specific)  `bracketSameLine`, which is available starting in Prettier 2.4.0. In this case, I guess we would want to use:
```
"bracketSameLine":true
```
We could maybe add both options and make it backwards compatible (not sure there could be conflicts, but in case it works, please add a comment indicating these details so we know we can safely remove it in the future). If that's not possible or we don't prefer it, we should just add the new `bracketSameLine`.

And my last question: do we consider these changes as `breaking` changes? Technically, if they reformat the existing documents in the Prettier context, I would say they are. So we should probably bump the major version. But this is debatable :)

Reply:
Sorry, I missed this commit an merged the pull request...
I'm going to fix it with https://github.com/oat-sa/prettier-config/pull/7

Reply:
Regarding the breaking change, I don't think it is, for it enforces the practices we followed since the beginning.

Reply:
Anyway, the following was done:
- remove the tag `0.1.2` and unpublish the version from npm
- remove the deprecated option `svelteBracketNewLine`
- set the option `bracketSameLine`
- declared a breaking change
See https://github.com/oat-sa/prettier-config/releases/tag/v1.0.0